homework9.1/task5.js

```diff
@@ -0,0 +1,15 @@
+// Створення масиву об'єктів users
+const users = [
+    { name: 'Іван', email: 'ivan@example.com', age: 25 },
+    { name: 'Марія', email: 'maria@example.com', age: 22 },
+    { name: 'Петро', email: 'petro@example.com', age: 30 }
+  ];
+  
+  // Використання циклу for...of для перебору елементів
+  for (const user of users) {
+    // Деструктуризація властивостей об'єкта user
+    const { name, email, age } = user;
```

Comment:
Я розумію що це було двома різними пунктами, але у тебе тут зайвий крок
Треба було зробити ось так:
```suggestion
    for (const { name, email, age } of user) {
```